apps/server/src/modules/tldraw/tldraw-test.module.ts

```diff
@@ -0,0 +1,29 @@
+import { DynamicModule, Module } from '@nestjs/common';
+import { User, Course } from '@shared/domain';
+import { MongoMemoryDatabaseModule } from '@shared/infra/database';
+import { MongoDatabaseModuleOptions } from '@shared/infra/database/mongo-memory-database/types';
+import { CoreModule } from '@src/core';
+import { LoggerModule } from '@src/core/logger';
+import { AuthenticationModule } from '@src/modules/authentication';
+import { AuthorizationModule } from '@src/modules/authorization';
+import { TldrawModule } from './tldraw.module';
+
+const imports = [
+	TldrawModule,
+	MongoMemoryDatabaseModule.forRoot({ entities: [User, Course] }),
+	AuthorizationModule,
```

Comment:
For the production module AuthorizationModule, AuthenticationModule do not exists for now. It will be added later, but then it should be added on both places.